paasta_tools/autoscaling/cluster_boost.py

```diff
@@ -89,6 +89,19 @@ def get_boosted_load(region: str, pool: str, current_load: float) -> float:
         return current_load
 
 
+def print_boost_value(region: str, pool: str) -> float:
+    """THis function prints the boost value if a boost is active
```

Comment:
NP: Fix capitalization

Reply:
It also isn't `print`ing the value, it is returning it.

Reply:
Good remark, I've renamed that function